yb-voyager/src/dbzm/dbzm.go

```diff
@@ -23,22 +22,18 @@ func NewDebezium(config *Config) *Debezium {
 	return &Debezium{Config: config}
 }
 
-func initVars() {
-	log.Infof("initializing debezium env variables like DEBEZIUM_DIST_DIR...")
+func init() {
```

Comment:
Move this function immediately after the declaration of the vars.

Reply:
ok